tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/modules/LiferayPortletModuleCustomPart.java

```diff
@@ -0,0 +1,58 @@
+/*******************************************************************************
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ *
+ *******************************************************************************/
+
+package com.liferay.ide.project.ui.modules;
+
+import org.eclipse.swt.widgets.Shell;
+
+/**
+ * @author Simon Jiang
+ */
+public class LiferayPortletModuleCustomPart extends LiferayPropertyModuleCustomPart
```

Comment:
Lets remove this class for now.  make the properties table a simple sapphire table that lets you add new rows with two columns, property key=value just like in other editors. since we don't have the information lets not show the dialog just make it a simple table and don't use a custom part.